.travis.yml

```diff
@@ -0,0 +1,23 @@
+sudo: false
+before_install:
+- curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.3.2
+- export PATH=$HOME/.yarn/bin:$PATH
+- gem install bundler -v '1.16.2'
+cache:
+  yarn: true
+language: ruby
+bundler_args: "--without development production staging"
+rvm:
+- 2.4.1
+addons:
+  postgresql: 9.6
+before_script:
+- psql -c 'create database travis_ci_test' -U postgres
+- cp config/database.yml.travis config/database.yml
+- cp config/secrets.yml.travis config/secrets.yml
+- yarn install
+- RAILS_ENV=test bundle exec rake db:migrate
+- RAILS_ENV=test bundle exec rails test
+notifications:
+  slack:
+    secure: K7YDsuNqaWmKeoseHArxd/BIvGXoCCItbh/m6JLZWNTR4K/pogd4SrbC1SHDH7RmSc2otM+HAU3Xl85F4CE61XsMiZQ9Vpz1LslsuwStoS3DO62lejwTRpVINfJS535stbUWPX8bNiU2OxHl2BLB+e3yeNUkiO9mFA2TbcfXti0Su0qwF7XFbD6txEWrJ7s0Ix9hhDOfUyfVjjLMu5cVQUkqeT6jICN+zdA85qduBmiWRgz2KuhD4SYqz4Yrn/4LEZEnw/C54dlHM13p1/ZAO6ypfC6EUDT3T7s97CWiUHyby5zI7dMt2BR0P/P/SR5isg8v5OYDKri8/qKzzKC5jeejdYmuH8NlbMDnhdqKKAnbczljXVIfSyNDbuzB1DMu6ClvPm0sskOUH47Z7X2zLy4iYsPXA56vMU3WDGNnEG1N3V9Jw+phzSeYy5IV+P1kmsV8NRV9Ch000Cc8lyrhk8C1Me6CftUveZAxLaOca1TFya5eyBsw5YEecmx6RyKF38IbMazFfs/EkATHZxZE1EjZGKTTDtIjm0OTxQQAe43k8m6NJIOY6regZ8GoJagvp+TPftyKAQQkclymqYBxF6jkR4VKtgybAr2FWmu0NGgY6psGij7TFMz1yRutTIqeml2USlUjEUajHxVCLFnUuBPC6dEKtTIRuVFagxfIT/A=
```

Comment:
Are there any security concerns on having this key publicly available here?

Reply:
@Levia this is supposed to be the secure way of doing it, as far as I know from the docs 🤷‍♂ please advise if I have made a mistake here!

Reply:
That's cool! It looks like it's fine having it here, I've double checked

Reply:
Cool thanks!